playground/cli/simple-pdf-to-json/newjson2pdf.js

```diff
@@ -0,0 +1,70 @@
+//'{"name": "Naval","twitterHandle": "naval", "city":"New York","image":"abs.jpg","tweet": "Tweet1","createAt": "2018-05-31"}')
+
+//parse json before using it as data
+var data = [
+  {
+    name: "Adam ",
+    twitterHandle: "@horse",
+    createdAt: "2018-05-31",
+    tweet: "How to Get Rich (without getting lucky) 1",
+    image: "http://placekitten.com/g/200/300",
+  },
+  {
+    name: "Kate ",
+    twitterHandle: "@cat",
+    createdAt: "2018-05-31",
+    tweet: "How to Get Rich (without getting lucky) 2",
+    image: "http://placekitten.com/g/200/300",
+  },
+  {
+    name: "Jenny ",
+    twitterHandle: "@squirrel",
+    createdAt: "2018-05-31",
+    tweet: "How to Get Rich (without getting lucky) 3",
+    image: "http://placekitten.com/g/200/300",
+  },
+];
+function display(_tweet) {
+  let myUl = document.querySelector("ul");
+  data.forEach(function (obj) {
```

Comment:
This code may be more readable if it looks like this, `tweets.forEach(function (tweet) {`. What do you think?

Reply:
@tr0mbl3y @johnjacobkenny 
Let's try to use ES6 format everywhere for more clean code.